src/site/content/en/payments/orchestrating-payment-transactions-with-a-service-worker/index.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: post
+title: Orchestrating payment transactions with a service worker
+subhead: |
+  How to adapt your web-based payment app to Web Payments and provide a better user experience for customers.
+authors:
+  - agektmr
+date: 2020-08-14
+description: |
+  Once a web-based payment app is registered, it's ready to accept payment requests from merchants. This article teaches you how to orchestrate a payment transaction from a service worker during runtime.
+tags:
+  - payments
```

Comment:
```suggestion
  - blog
  - payments
  - service-worker
  - monetization
```
I just noticed that we have a [monetization tag](https://web.dev/tags/monetization/). LMK if we should open an issue and figure out what to do with it, or if we should just keep `payments` and `monetization` (and perhaps add `monetization` to all pages in the web.dev/payments collection)

Reply:
I feel adding `monetization` tag to this article isn't suitable. The tag is more for merchants. This article is for payment handlers and doesn't contribute to monetization directly.

Reply:
Ah, that makes sense